### DISCUSSION TITLE
web access (on port 5000) for latest release on Home assistant removed?

### DISCUSSION THREAD

**Discussion by rugene76** [blakeblackshear/frigate#3939] [General]
I'm wondering if the new frigate release eliminated web access on port 5000 when installed as a plugin on home assistant?
Previously I could access frigate locally at http://ha_ipaddress:5000. Now that same address fails.

I've just completed the upgrade with several difficulties, e.g., had to uninstall/install. Frigate (latest) does seem to be operating
properly when accessed within home assistant's interface. 
Is there some setting I'm missing?
thanks
(btw, all related software is at latest release, running on intel hardware)

**Comment by NickM-27** [comment]
~~No, it is the same as always. Make sure you don't have another container also using port 5000~~

**Reply by rugene76** [reply]
turns out I had to hard code it in the config page. I had thought it was set by default.

**Comment by blakeblackshear** [comment]
The latest addon version doesn't expose port 5000 by default anymore since it does not have any authentication. You must set the port in the addon config now.

**Reply by vjmaz** [reply]
Sorry Blake how do I added this in the addon config please

**Comment by vjmaz** [comment]
Interesting I added them in the Show disabled ports in the addon and it ask to restart and it worked Cool 
I have added a screen shot if it works so others can try too :) hope it works for use all 

![image](https://github.com/blakeblackshear/frigate/assets/81817593/b2b4ccb6-82da-4b38-be64-6a31904287e0)

**Comment by tung256** [comment]
i opened all my ports
![image](https://github.com/user-attachments/assets/daa1ef20-c966-443e-9ece-87c056057b48)

i still get errors.
any idea how to fix?
![image](https://github.com/user-attachments/assets/ec005091-083b-45fe-bdb5-c7490593d9a0)

**Reply by NickM-27** [reply]
you need to use the ip address when the port is opened

**Reply by tung256** [reply]
@NickM-27 
still no luck. HA add-on is on my HA OS with ip of 192.168.1.229
![image](https://github.com/user-attachments/assets/09fe44cd-8dd7-41ce-95d0-23651deeee82)
i entered in the IP address and port as suggested but not working. is there anything i have to do with HA's config yml file to get into that port?

**Reply by NickM-27** [reply]
are you sure the addon is up and running?

**Reply by tung256** [reply]
@NickM-27  yes, very sure it is running. i got v12 working as you suggested
![image](https://github.com/user-attachments/assets/b5abda9a-3473-4eb7-8714-ab0e50a2dbce)

![image](https://github.com/user-attachments/assets/54e3d4fc-5a8c-49cd-a448-d45251f6febd)

the HACS integration is actually v 1.1.1 from 2021 since that was working with my v12 before i updated to v14.
![image](https://github.com/user-attachments/assets/229a9ad6-3d97-49b2-b7f7-a071b8867623)
do you think the latest hacs 5.4.0 is compatible with v12 for me to try?

**Comment by tung256** [comment]
@NickM-27 i just started Frigate hass-add-on v14 again and it works great with hacs frigate 5.4.0. no issues.
BUT whenever i open the Frigate Web UI on Firefox or Edge, the add-on would stop within 10 seconds....
here is my Frigate yml file https://drive.google.com/file/d/1yAYZ-LXFSuUq12v3sNsLowDRJAHYbDZo/view?usp=sharing
and here is the error log: https://drive.google.com/file/d/16pTXskYFmJG0K9gpQMJStDHXl86l7_XM/view?usp=sharing
any idea why?
OR is there a bare minimum config file for Frigate to work just for detection? basically, i am using it with the Coral chip for person detection to turn lights On/Off. im not actually using Frigate to record videos or anything crazy

**Reply by NickM-27** [reply]
can see your camera is saying the username / password is wrong 
`2024-10-03 12:27:27.092096661  [2024-10-03 08:27:27] ffmpeg.garage.detect           ERROR   : rtsp://*:*@192.168.1.149:554/stream1: Server returned 401 Unauthorized (authorization failed)
`

**Reply by tung256** [reply]
yes, i noticed that too. very strange that it works in VLC with that RTSP address....
![image](https://github.com/user-attachments/assets/b71c51e9-50aa-424e-ac42-5044beb03f0b)
i copied and pasted the id and pass from frigate.yml directly to VLC to be extra sure.

**Reply by NickM-27** [reply]
it could be an issue if it has some special characters
